qcodes_contrib_drivers/drivers/HP/HP8594E.py

```diff
@@ -0,0 +1,207 @@
+import logging
+from typing import Any, Dict
+import struct
+import numpy as np
+import datetime
+import qcodes.utils.validators as vals
+from qcodes.instrument.parameter import (
+    Parameter,
+    ParameterWithSetpoints,
+    ParamRawDataType,
+)
+from qcodes.instrument.visa import VisaInstrument
+import numpy.typing as npt
+
+log = logging.getLogger(__name__)
```

Comment:
I can't see that this logger is used anywhere?

Reply:
removed unused logger